File: 2529-maximum-count-of-positive-integer-and-negative-integer/2529. Maximum Count of Positive Integer and Negative Integer.java
class Solution {
    public int maximumCount(int[] nums) {

        // int n = nums.size();
        int pos = 0;
        int neg = 0;

        for(Integer it : nums)
        {
            if(it>0)pos++;
            if(it < 0)neg++;
        }

        return Math.max(pos, neg);
        
    }
}

File: 2529-maximum-count-of-positive-integer-and-negative-integer/2529. Maximum Count of Positive Integer and Negative Integer.cpp
class Solution {
public:
    int maximumCount(vector<int>& nums) 
    {
    int ncount=0;
    int pcount=0;
    for(int i=0;i<nums.size();i++){
        if(nums[i]>0)
        pcount++;
        
        if(nums[i]<0)
        ncount++;
    }   
    int maxvalue=max(ncount,pcount);
    return maxvalue;
    }
};